lib/sink/connection/server_handler.ex

```diff
@@ -242,6 +242,12 @@ defmodule Sink.Connection.ServerHandler do
               register_when_clear(client_id)
           end
 
+        if ConnectionStatus.connected?(state.connection_status) do
+          # Fake connection response to handle
+          # TODO: Remove once connection requests are required, as the condition will never be true
+          handler.handle_connection_response(state.client, :connected)
```

Comment:
The callback will be called twice for clients making connection requests, but that should be fine, as all it does is update the `last_connected_at`.